src/app/state/organization/utils/get-main-orgs.ts

```diff
@@ -0,0 +1,40 @@
+export const getMainOrgs = ($getOrgList: any[]) => {
+  const list = [];
+  const subList = [];
+
+  // Separate Parent & Child Orgs
+  $getOrgList.forEach((item: any) => {
+    const namespaceArr = item.namespace.split('.');
+    if (namespaceArr.length === 3) {
+      list.push(item);
+    } else {
+      if (!subList[namespaceArr.length - 4]) {
+        subList[namespaceArr.length - 4] = [];
+      }
+      subList[namespaceArr.length - 4].push(item);
+    }
+  });
```

Comment:
The `3` and `4` are a bit of a magic numbers. I think constants should be used.
I'm not sure why the `4`. If you had a suborg of `finance.dawidcorp.iam.ewc`, `namespaceArr.length` would be `4`. So I guess what you're trying to do is get lists of the suborg that are at each "level"?
```suggestion
  // Separate Parent & Child Orgs
   const numberOfLabelsInOrgNamespace = 3;
  $getOrgList.forEach((item: any) => {
    const namespaceArr = item.namespace.split('.');
    if (namespaceArr.length === numberOfLabelsInOrgNamespace) {
      list.push(item);
    } else {
      if (!subList[namespaceArr.length - 4]) {
        subList[namespaceArr.length - 4] = [];
      }
      subList[namespaceArr.length - 4].push(item);
    }
  });
```